src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,10 @@
+package org.example;
+
+public class Main {
+    public static void main(String[] args) {
+        System.out.println("Hello world!");
+        //PaySlip paySlip = new PaySlip(10,20,30,20,70);
+        PaySlip paySlip = new PayslipBuilder().setBasic(20).setDa(90).build();
```

Comment:
Can you think of a way how this can handle the presence of _required_ and _optional_ fields?